cmd/archeio/main_test.go

```diff
@@ -33,8 +33,6 @@ import (
 // TestIntegrationMain tests the entire, built binary with an integration
 // test, pulling images with crane
 func TestIntegrationMain(t *testing.T) {
-	integration.MaybeSkip(t)
```

Comment:
this was a hold-over from KIND where we iterated on this previously. the test tags and regex matching should be sufficient

Reply:
ack